tools/gbench/report.py

```diff
@@ -143,32 +146,43 @@ def extract_field(partition, field_name):
     return [lhs, rhs]
 
 
-def print_utest(partition, utest_alpha, first_col_width, use_color=True):
-    timings_time = extract_field(partition, 'real_time')
-    timings_cpu = extract_field(partition, 'cpu_time')
-
+def calc_utest(timings_cpu, timings_time):
```

Comment:
I think these `print_utest()` changes can count as a separate preparatory NFC refactoring,
could you please split this into separate PR?

Reply:
Done. #740